zowe/zowe-cli-secrets-for-kubernetes
# VSCE: Add extension icon

**What It Does**

Adds an LFX approved icon to the extension

**How to Test**


You can run the `npm run package` command and inspect the VSIX 😋 
<img width="916" alt="image" src="https://github.com/user-attachments/assets/adef3974-a6c9-441a-b40b-a7bab2cab438">

Once installed, you may see the icon in the list of installed extensions
<img width="322" alt="image" src="https://github.com/user-attachments/assets/5c30d93b-e41a-44d8-8cb5-d628c9629dae">


**Review Checklist**
I certify that I have:
